sw/device/lib/testing/test_framework/ottf_main.h

```diff
@@ -107,4 +109,20 @@ void ottf_task_delete_self(void);
  */
 char *ottf_task_get_self_name(void);
 
+/**
+ * Execute a test function and log the test result.  Update the result value
+ * if there is a failure code.
+ */
+#define EXECUTE_TEST(result_, test_function_)                            \
```

Comment:
nit: maybe this should go in `sw/device/lib/testing/test_framework/ottf_macros.h`?

Reply:
Agreed, I would prefer just including `ottf_macros.h` in silicon_creator files instead of `ottf_main.h`.

Reply:
Ack, thanks @cfrantz!